python/tests/sklearn/preprocessing/data_test.py

```diff
@@ -369,68 +368,6 @@ def label_encoder_deserializer_test(self):
         self.assertEqual(le.input_features, label_encoder_tf.input_features)
         self.assertEqual(le.output_features, label_encoder_tf.output_features[0])
 
-    def one_hot_encoder_serializer_test(self):
```

Comment:
These tests were moved to `python/tests/sklearn/preprocessing/one_hot_encoder_test.py`